input/chains/penumbra-1.json

```diff
@@ -101,7 +101,84 @@
       ]
     }
   ],
-  "canonicalNumeraires": [
-    "transfer/channel-2/uusdc"
-  ]
+  "canonicalNumeraires": ["transfer/channel-2/uusdc"],
+  "priorityScoresByBase": {
+    "transfer/channel-4/factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/COOK": 1,
+    "transfer/channel-4/factory/osmo1dywfmhyc8y0wga7qpzej0x0mgwqg25fj4eccp494w8yafzdpgamsx9ryyv/fSLOTH": 1,
+    "transfer/channel-4/factory/osmo1vdvnznwg597qngrq9mnfcfk0am9jdc9y446jewhcqdreqz4r75xq5j5zvy/ymos": 1,
+    "transfer/channel-4/factory/osmo1pfyxruwvtwk00y8z06dh2lqjdj82ldvy74wzm3/WOSMO": 14,
+    "transfer/channel-4/factory/osmo19hdqma2mj0vnmgcxag6ytswjnr8a3y07q7e70p/wLIBRA": 1,
+    "transfer/channel-4/factory/osmo14mafhhp337yjj2aujplawz0tks6jd2lel4hkwz4agyzhvvztzaqsqzjq8x/alloyed/allTRX": 18,
+    "transfer/channel-4/factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/cac": 1,
```

Comment:
question: Is there a reason why we would add a priority score of 1 to any? Feels like we can drop it entirely if it's that low